proposals/blogProposal.md

```diff
@@ -0,0 +1,11 @@
+The Web of Things standardization activities are now {illustrated|explained} through a new animation video that we have been working on for the last few months. You can see how any web developer can implement Internet of Things solutions while being interoperable and not locked into frameworks. Additionally, the W3C WoT allows the experience of the developers to be transferred to different application domains such as agriculture, building or industrial automation, smart home and more!
+
+-- To be decided
+
+We would like to thank the W3C Web Accessibility Initiative team for their feedback, as well as Siemens AG and Intel Inc. for the support and the team at WebCastor for quick iteration cycles to come up with our amazing video! 
```

Comment:
we should remove the names of the companies.

Reply:
"Initiative team for their feedback, as well as Siemens AG and Intel Inc. for the ..." --> "Initiative team for their feedback and the ..."